bin/nyc.js

```diff
@@ -117,7 +117,7 @@ if (process.env.NYC_CWD) {
   } else if (argv._.length) {
     // wrap subprocesses and execute argv[1]
     var nyc = (new NYC())
-    nyc.cleanup()
+    nyc.reset()
```

Comment:
> The issue with all was that it was calling cleanup(), which deleted directories, but did not recreate them.

Where did it call `cleanup()`? That's the part I'm not understanding, I suppose.



> I just wanted to avoid an additional change creeping into this PR.

Sure.